src/WPF/WPF.Viewer/Samples/GraphicsOverlay/CreateAndEditGeometries/CreateAndEditGeometries.xaml

```diff
@@ -0,0 +1,211 @@
+﻿<UserControl x:Class="ArcGIS.WPF.Samples.CreateAndEditGeometries.CreateAndEditGeometries"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:esri="http://schemas.esri.com/arcgis/runtime/2013">
+    <Grid>
+        <esri:MapView x:Name="MyMapView" GeoViewTapped="MyMapView_GeoViewTapped" />
+        <Border Grid.Column="1"
+                Margin="10"
+                Padding="10"
+                HorizontalAlignment="Right"
+                VerticalAlignment="Top"
+                Background="White">
+            <Grid>
+                <Grid.RowDefinitions>
+                    <RowDefinition Height="auto" />
+                    <RowDefinition Height="auto" />
+                    <RowDefinition Height="auto" />
+                    <RowDefinition Height="auto" />
+                    <RowDefinition Height="auto" />
+                    <RowDefinition Height="auto" />
+                    <RowDefinition Height="auto" />
+                    <RowDefinition Height="auto" />
+                    <RowDefinition Height="auto" />
+                    <RowDefinition Height="auto" />
+                </Grid.RowDefinitions>
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="75" />
+                    <ColumnDefinition Width="75" />
+                </Grid.ColumnDefinitions>
+                <Label Grid.ColumnSpan="2"
+                       HorizontalAlignment="Center"
+                       Content="Create"
+                       FontSize="20"
+                       FontWeight="Bold" />
+                <Button x:Name="PointButton"
+                        Grid.Row="1"
+                        Click="PointButton_Click">
+                    <Button.ContentTemplate>
+                        <DataTemplate>
+                            <StackPanel>
+                                <TextBlock HorizontalAlignment="Center"
```

Comment:
When you find several properties tend to repeat, I think it's good indication to introduce a style.

```XML
    <UserControl.Resources>
        <Style x:Key="IconStyle"
               TargetType="TextBlock">
            <Style.Setters>
                <Setter Property="HorizontalAlignment"
                        Value="Center" />
                <Setter Property="FontFamily"
                        Value="pack://application:,,,/Resources/Fonts/#calcite-ui-icons-24" />
                <Setter Property="FontSize"
                        Value="40" />
            </Style.Setters>
        </Style>
    </UserControl.Resources>

    <!-- more code here -->
                                <TextBlock Style="{StaticResource IconStyle}"
                                           Text="&#xe21c;" />

 
```

Reply:
Agree with Jennifer, I think the xaml just now is slightly too verbose and hard to follow.